packages/portal/src/components/iiif/IIIFViewer.vue

```diff
@@ -720,13 +583,13 @@
       },
 
       findDownloadLinkForPage2(pageId) {
```

Comment:
not related to this PR specifically, but we should rethink the way these method names are constucted, especially on this and the `findDownloadLinkForPage3` method the name is potentially confusing.

`findDownloadLinkForPageV3` & `findDownloadLinkForPageV2` would be clearer.

Reply:
Agreed, done.